eq-author/src/App/section/Design/index.test.js

```diff
@@ -136,6 +136,9 @@ describe("SectionRoute", () => {
 
   beforeEach(() => {
     childContextTypes = { router: PropTypes.object };
+    const toasts = document.createElement("div");
```

Comment:
Do we need a Toast div? There's no reference to toast in Design/index.js now

Reply:
Yes, unfortunately using the TestProvider relies on it :(